vllm/v1/engine/output_processor.py

```diff
@@ -248,22 +267,22 @@ def process_outputs(
         1) Compute stats for logging
         2) Detokenize
         3) Create and handle RequestOutput objects:
-            * If there is a queue (for usage with AsyncLLM), 
+            * If there is a queue (for usage with AsyncLLM),
               put the RequestOutput objects into the queue for
               handling by the per-request generate() tasks.
 
-            * If there is no queue (for usage with LLMEngine), 
```

Comment:
Can you add a comment to the docstring about why we finish the stop string requests externally to this function?

Reply:
done